roles/barbican/templates/etc/barbican/barbican.conf

```diff
@@ -0,0 +1,133 @@
+# {{ ansible_managed }}
+
+[DEFAULT]
+
+use_stderr = False
+debug = False
+
+bind_host = 0.0.0.0
+
+bind_port = {{ endpoints.barbican.port.backend_api }}
+
+host_href = {{ endpoints.barbican.url.public }}
+
+log_file = /var/log/barbican/api.log
+
+backlog = 4096
```

Comment:
should parameterize few tunnable attributes and set default values in role defaults.
here are list of attributes we should parameterize:
backlog
max_allowed_secret_in_bytes
max_allowed_request_size_in_bytes
sql_idle_timeout
default_limit_paging
max_limit_paging

asynchronous_workers

initial_delay_seconds
periodic_interval_max_seconds

thread_pool_size